--- ./work/qtmultimedia/src/plugins/common/evr/evrcustompresenter.cpp.orig	2017-09-20 07:26:10.592952400 -0700
+++ ./work/qtmultimedia/src/plugins/common/evr/evrcustompresenter.cpp	2017-09-20 07:29:01.525710400 -0700
@@ -2027,8 +2027,10 @@
         return QVideoFrame::Format_RGB555;
     else if (subtype == MFVideoFormat_AYUV)
         return QVideoFrame::Format_AYUV444;
+#if defined(_MSC_VER) && (_MSC_VER >= 1900)
     else if (subtype == MFVideoFormat_I420)
         return QVideoFrame::Format_YUV420P;
+#endif
     else if (subtype == MFVideoFormat_UYVY)
         return QVideoFrame::Format_UYVY;
     else if (subtype == MFVideoFormat_YV12)
