From 75d9cc5949e632daf61fd2a6040059984d31a10e Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Fri, 16 Feb 2018 12:19:50 +0000
Subject: [PATCH 2/2] macOS 10.9: Reinstate support

---
 chromium/base/mac/mac_util.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/qtwebengine/src/3rdparty/chromium/base/mac/mac_util.h b/qtwebengine/src/3rdparty/chromium/base/mac/mac_util.h
index 37e5b670ce..da1d4d3e8b 100644
--- a/qtwebengine/src/3rdparty/chromium/base/mac/mac_util.h
+++ b/qtwebengine/src/3rdparty/chromium/base/mac/mac_util.h
@@ -133,7 +133,12 @@ BASE_EXPORT int MacOSXMinorVersion();
 #define IGNORE_DEPLOYMENT_TARGET(OP, V, RET)
 
 DEFINE_IS_OS_FUNCS(9, TEST_DEPLOYMENT_TARGET)
+
+#ifdef MAC_OS_X_VERSION_10_10
 DEFINE_IS_OS_FUNCS(10, TEST_DEPLOYMENT_TARGET)
+#else
+DEFINE_IS_OS_FUNCS(10, IGNORE_DEPLOYMENT_TARGET)
+#endif
 
 #ifdef MAC_OS_X_VERSION_10_11
 DEFINE_IS_OS_FUNCS(11, TEST_DEPLOYMENT_TARGET)
-- 
2.15.0

