--- ./work/qtwebengine/src/core/config/common.pri.orig	2019-01-30 10:06:04.776357953 +0000
+++ ./work/qtwebengine/src/core/config/common.pri	2019-01-30 09:46:04.170282106 +0000
@@ -2,8 +2,12 @@
 include($$QTWEBENGINE_OUT_ROOT/src/core/qtwebenginecore-config.pri)
 QT_FOR_CONFIG += webenginecore
 
+# This is horrible, sorry
+CONDA_PREFIX=$$dirname(QMAKE_INCDIR_WEBENGINE-ZLIB)
+
 gn_args += \
     use_qt=true \
+    conda_prefix=\"$$CONDA_PREFIX\" \
     is_component_build=false \
     is_shared=true \
     enable_message_center=false \
--- ./work/qtwebengine/src/3rdparty/chromium/build/linux/unbundle/zlib.gn.orig	2019-01-30 00:36:42.021657473 +0000
+++ ./work/qtwebengine/src/3rdparty/chromium/build/linux/unbundle/zlib.gn	2019-01-30 00:48:00.523433133 +0000
@@ -11,6 +11,8 @@
 
 config("system_zlib") {
   defines = [ "USE_SYSTEM_ZLIB=1" ]
+  include_dirs = [ conda_prefix+"/include" ]
+  lib_dirs = [ conda_prefix+"/lib" ]
 }
 
 source_set("zlib") {
--- ./work/qtwebengine/src/3rdparty/chromium/third_party/pdfium/third_party/BUILD.gn.orig	2019-01-30 15:01:16.272785985 +0000
+++ ./work/qtwebengine/src/3rdparty/chromium/third_party/pdfium/third_party/BUILD.gn	2019-01-30 15:01:10.137017538 +0000
@@ -8,6 +8,10 @@
 import("//build_overrides/build.gni")
 import("../pdfium.gni")
 
+declare_args() {
+  conda_prefix = ""
+}
+
 group("third_party") {
   deps = [
     ":bigint",
@@ -328,6 +332,8 @@
 config("system_zlib_config") {
   libs = [ "z" ]
   defines = [ "USE_SYSTEM_ZLIB" ]
+  include_dirs = [ conda_prefix+"/include" ]
+  lib_dirs = [ conda_prefix+"/lib" ]
 }
 
 group("zlib") {
