--- qtbase/mkspecs/common/clang.conf.orig	2018-03-13 14:00:46.000000000 -0700
+++ qtbase/mkspecs/common/clang.conf	2018-03-13 14:19:20.000000000 -0700
@@ -21,6 +21,7 @@
 QMAKE_CFLAGS_USE_PRECOMPILE      = -Xclang -include-pch -Xclang ${QMAKE_PCH_OUTPUT}
 QMAKE_CFLAGS_LTCG                = -flto
 QMAKE_CFLAGS_DISABLE_LTCG        = -fno-lto
+QMAKE_CFLAGS_OPTIMIZE_SIZE       = -Oz -fomit-frame-pointer -fdata-sections -fvisibility=hidden
 
 QMAKE_CXXFLAGS_PRECOMPILE        = -x c++-header -c ${QMAKE_PCH_INPUT} -o ${QMAKE_PCH_OUTPUT}
 QMAKE_CXXFLAGS_USE_PRECOMPILE    = $$QMAKE_CFLAGS_USE_PRECOMPILE
