diff --git a/mkspecs/common/clang.conf b/mkspecs/common/clang.conf
index dad15a22a8..4f0de202f7 100644
--- a/mkspecs/common/clang.conf
+++ b/mkspecs/common/clang.conf
@@ -16,7 +16,7 @@ QMAKE_LINK_SHLIB        = $$QMAKE_CXX
 CONFIG                 += clang_pch_style
 QMAKE_PCH_OUTPUT_EXT    = .pch

-QMAKE_CFLAGS_OPTIMIZE_SIZE = -Oz
+QMAKE_CFLAGS_OPTIMIZE_SIZE = -Oz -fomit-frame-pointer -fdata-sections -fvisibility=hidden

 QMAKE_CFLAGS_PRECOMPILE          = -x c-header -c ${QMAKE_PCH_INPUT} -o ${QMAKE_PCH_OUTPUT}
 QMAKE_CFLAGS_USE_PRECOMPILE      = -Xclang -include-pch -Xclang ${QMAKE_PCH_OUTPUT}
