diff --git a/qtbase/configure.json b/qtbase/configure.json
index c273aeeef1..02b0d0c4d8 100644
--- a/qtbase/configure.json
+++ b/qtbase/configure.json
@@ -415,7 +415,7 @@
         "enable_new_dtags": {
             "label": "new dtags support",
             "type": "linkerSupportsFlag",
-            "flag": "--enable-new-dtags"
+            "flag": "--disable-new-dtags"
         },
         "gdb_index": {
             "label": "gdb index support",
diff --git a/qtbase/mkspecs/common/gcc-base-unix.conf b/qtbase/mkspecs/common/gcc-base-unix.conf
index a456c8f3eb..5e21ad899c 100644
--- a/qtbase/mkspecs/common/gcc-base-unix.conf
+++ b/qtbase/mkspecs/common/gcc-base-unix.conf
@@ -18,7 +18,7 @@ QMAKE_LFLAGS_SONAME    += -Wl,-soname,
 QMAKE_LFLAGS_THREAD    +=
 QMAKE_LFLAGS_RPATH      = -Wl,-rpath,
 QMAKE_LFLAGS_RPATHLINK  = -Wl,-rpath-link,
-QMAKE_LFLAGS_NEW_DTAGS  = -Wl,--enable-new-dtags
+QMAKE_LFLAGS_NEW_DTAGS  = -Wl,--disable-new-dtags
 QMAKE_LFLAGS_GDB_INDEX  = -Wl,--gdb-index
 QMAKE_LFLAGS_USE_BFD    = -fuse-ld=bfd
 QMAKE_LFLAGS_USE_GOLD   = -fuse-ld=gold
