From d2f9bb4ebfb9f6f935a5d5f8dcd2c5dba628f307 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Tue, 23 Jun 2020 22:17:19 +0200
Subject: [PATCH] qtwebengine: Add conda_prefix to gn

---
 src/buildtools/config/common.pri | 1 +
 1 file changed, 1 insertion(+)

diff --git a/qtwebengine/src/buildtools/config/common.pri b/qtwebengine/src/buildtools/config/common.pri
index 9824897f..d554313b 100644
--- a/qtwebengine/src/buildtools/config/common.pri
+++ b/qtwebengine/src/buildtools/config/common.pri
@@ -5,6 +5,7 @@ QT_FOR_CONFIG += buildtools-private webenginecore webenginecore-private
 
 gn_args += \
     use_qt=true \
+    conda_prefix=\"$$CONDA_PREFIX\" \
     closure_compile=false \
     is_component_build=false \
     is_shared=true \
-- 
2.27.0

